src/fluid/main.cpp

```diff
@@ -45,27 +45,29 @@ constexpr int SCALE{3};
 class FluidSimulationApp : public Magnum::Platform::Application {
  public:
   FluidSimulationApp(const Arguments& arguments)
-  : Magnum::Platform::Application{
-      arguments,
-      Configuration{}.setTitle("Codeplay Fluid Simulation"
-        " - Move mouse to add fluid - Press space to clear fluid"),
-      GLConfiguration{}.setFlags(GLConfiguration::Flag::QuietLog)},
-    size_{SIZE},
-    fluid_{SIZE, 0.2f, 0.0f, 0.0000001f},
-    mesh_{Magnum::MeshTools::compile(Magnum::Primitives::squareSolid(
-      Magnum::Primitives::SquareFlag::TextureCoordinates))},
-    shader_{Magnum::Shaders::FlatGL2D::Configuration{}.setFlags(
-      Magnum::Shaders::FlatGL2D::Flag::Textured |
-      Magnum::Shaders::FlatGL2D::Flag::TextureTransformation)} {
-
+      : Magnum::Platform::Application{arguments,
+                                      Configuration{}.setTitle(
+                                          "Codeplay Fluid Simulation"
+                                          " - Move mouse to add fluid - Press "
+                                          "space to clear fluid"),
```

Comment:
I know this is a code change, but maybe it's worth it to make this a constexpr char* or something to fix the formatting here, because it's not very readable scrolling off the side like this

Reply:
Moved to `constexpr std::string_view WINDOWTITLE`